dipy/tracking/direction_getter.pyx

```diff
@@ -1,11 +1,141 @@
+cimport cython
+from .direction_getter cimport DirectionGetter
```

Comment:
I do not think this import is needed. It is strange here to import itself 😄

Reply:
yes... removed